drivewise/src/Components/Nav Bar/NavBar.js

```diff
@@ -0,0 +1,105 @@
+import * as React from 'react';
+import AppBar from '@mui/material/AppBar';
+import Box from '@mui/material/Box';
+import Toolbar from '@mui/material/Toolbar';
+import IconButton from '@mui/material/IconButton';
+import Typography from '@mui/material/Typography';
+import Menu from '@mui/material/Menu';
+import Container from '@mui/material/Container';
+import Avatar from '@mui/material/Avatar';
+import Button from '@mui/material/Button';
+import Tooltip from '@mui/material/Tooltip';
+import MenuItem from '@mui/material/MenuItem';
+import Logo from '../Logo/Logo';
+
+const pages = ['Home', 'About', 'Contact Us'];
+const settings = ['Account', 'Event Dashboard', 'Logout'];
+
+function ResponsiveAppBar() {
+  const [anchorElUser, setAnchorElUser] = React.useState(null);
+
+
+  const handleOpenUserMenu = (event) => {
+    setAnchorElUser(event.currentTarget);
+  };
+
+
+
+  const handleCloseUserMenu = () => {
+    setAnchorElUser(null);
+  };
+
+  return (
+    <>
+      <AppBar position="fixed" style={{ background: '#f96d6d' }}>
```

Comment:
Try and use the free Tailwind UI components if you can for these. If not MUI is fine.